src/components/common/Extensions/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable max-len */
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import { useIntl } from 'react-intl';
+import { CheckboxProps } from './types';
```

Comment:
Add blank space between imports packages and other tools like types, components, hooks etc.